test/core_tests.html

```diff
@@ -2,39 +2,22 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>QUnit tests for impress.js</title>
+  <title>Core Tests</title>
   <link rel="stylesheet" href="qunit/qunit.css">
 </head>
 
-
-
-<body class="impress-not-supported">
-  <!-- HTML elements to be tested with QUnit should typically be inside
-       a qunit-fixture div. Unfortunately that just won't work with impress.js
-       as it wants the div#impress element to be a direct child of body. Or
-       something, I don't know, but it didn't work.
-       
-       So we just have an empty qunit-fixture, and then our impress steps
-       just below it. This just means you have to make sure that each
-       QUnit.test() cleans up after itself, can't rely on QUnit to do it now.
-  -->
-  <div id="qunit-fixture">
```

Comment:
This is because there are several global states and events that impress.js uses, they should be unregistered from body/window if one is supposed to clean everything (QUnit just removes the HTML, not the attached events).